include/aspect/boundary_traction/interface.h

```diff
@@ -126,31 +126,172 @@ namespace aspect
         const GeometryModel::Interface<dim> *geometry_model;
     };
 
+    template <int dim>
+    class Manager : public ::aspect::SimulatorAccess<dim>
+    {
+      public:
+        /**
+         * Destructor. Made virtual since this class has virtual member
+         * functions.
+         */
+        ~Manager () override;
+
+        /**
+         * A function that is called at the beginning of each time step and
+         * calls the corresponding functions of all created plugins.
+         *
+         * The point of this function is to allow complex boundary traction
+         * models to do an initialization step once at the beginning of each
+         * time step. An example would be a model that needs to call an
+         * external program to compute the traction change at a boundary.
+         */
+        virtual
+        void
+        update ();
+
+        /**
+         * A function that calls the boundary_traction functions of all the
+         * individual boundary traction objects and uses the stored operators
+         * to combine them.
+         */
+        Tensor<1,dim>
+        boundary_traction (const types::boundary_id boundary_indicator,
+                           const Point<dim> &position,
+                           const Tensor<1,dim> &normal_vector) const;
 
+        /**
+         * A function that is used to register boundary traction objects in such
+         * a way that the Manager can deal with all of them without having to
+         * know them by name. This allows the files in which individual
+         * plugins are implemented to register these plugins, rather than also
+         * having to modify the Manager class by adding the new boundary
+         * traction plugin class.
+         *
+         * @param name A string that identifies the boundary traction model
+         * @param description A text description of what this model does and that
+         * will be listed in the documentation of the parameter file.
+         * @param declare_parameters_function A pointer to a function that can be
+         * used to declare the parameters that this boundary traction model
+         * wants to read from input files.
+         * @param factory_function A pointer to a function that can create an
+         * object of this boundary traction model.
+         */
```

Comment:
This documentation does not belong to this function. Delete it and copy over the documentation of the same function from the boundary velocity manager and adjust for traction boundaries.